packages/core/injected-metadata/core-injected-metadata-common-internal/src/types.ts

```diff
@@ -53,10 +64,7 @@ export interface InjectedMetadata {
   i18n: {
     translationsUrl: string;
   };
-  theme: {
-    darkMode: boolean;
-    version: ThemeVersion;
-  };
+  theme: InjectedMetadataTheme;
```

Comment:
I guess InjectedMetadata will stay with us for a while longer

Reply:
True, but fwiw this PR isn't introducing anything new that wasn't already in the list of things we need to figure out to get rid of this injection system.